src/pages/settings/InitialSettingsPage.tsx

```diff
@@ -187,39 +178,36 @@ function InitialSettingsPage(props) {
         };
 
         if (NativeModules.HybridAppModule) {
-            const hybridAppMenuItems = _.filter(
-                [
-                    {
-                        translationKey: 'initialSettingsPage.returnToClassic',
-                        icon: Expensicons.RotateLeft,
-                        shouldShowRightIcon: true,
-                        iconRight: Expensicons.NewWindow,
-                        action: () => NativeModules.HybridAppModule.closeReactNativeApp(),
-                    },
-                    ...defaultMenu.items,
-                ],
-                (item) => item.translationKey !== 'initialSettingsPage.signOut' && item.translationKey !== 'initialSettingsPage.goToExpensifyClassic',
-            );
+            const hybridAppMenuItems: MenuData[] = [
+                {
+                    translationKey: 'initialSettingsPage.returnToClassic' as const,
+                    icon: Expensicons.RotateLeft,
+                    shouldShowRightIcon: true,
+                    iconRight: Expensicons.NewWindow,
+                    action: () => NativeModules.HybridAppModule.closeReactNativeApp() as void,
```

Comment:
Why the assertion?

Reply:
Because this native module don't have a typescript support and the inferred type is `any`

Reply:
You can do like we did for `BootSplash` adding a module type like we have in `src/libs/BootSplash/types.ts`, and using it [here](https://github.com/Expensify/App/blob/main/src/types/modules/react-native.d.ts#L353).

Reply:
Oh right! resolved! @fabioh8010